index.html

```diff
@@ -15,11 +15,19 @@
             <nav class="menu-icon">
                 <img src="src/Icon-Menu.png" alt="menu"/>
             </nav>
+
+            <nav class="navDesk">
+                <ul class="navList">
+                  <li class="menuDesk"><a href="#works-section" class="menulink">Portfolio</a></li>
+                  <li class="menuDesk"><a href="#about-section">About</a></li>
+                  <li class="menuDesk"><a href="#contact-form">Contact</a></li>
+                </ul>
+            </nav> 
```

Comment:
- [OPTIONAL] Please add `background-size: contain` instead of `background-size: cover` for the hero section in the desktop version to match the Figma design.